jylee0805/momoCall
# Feature/update messages date

[Website URL](https://momocall--staging-chat-vj5jwzs8.web.app).

## Description
- Complete message date display in window

## Test Plan
1. Open the Website URL
2. Check message date display  different like “今天”、“剛剛”
